frontend/src/components/ProjectCardGrid/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect } from 'react'
+import { useInView } from 'react-intersection-observer'
+
+import { meiliIndex } from '@utils/meili'
+import { Filter } from 'meilisearch'
+import ProjectCard, { IntersectionObserverRef } from '../ProjectCard'
+import styles from './index.module.scss'
+
+export const cardsPerRow = 3
+const LIMIT = cardsPerRow * 6
```

Comment:
Why is LIMIT capitalized?

Reply:
Just to get around this https://github.com/kitspace/kitspace-v2/blob/9e229ddea33313dfc0b8d4d7d7bf10b713245cc3/frontend/src/components/ProjectCardGrid/index.tsx

I rename it to `defaultLimit`